### DISCUSSION TITLE
Animated Splash Screen

### DISCUSSION THREAD

**Discussion by VadimNotJustDev** [notJust-dev/notjustdev#157] [Ideas]
![image](https://github.com/notJust-dev/DEVember/assets/16936043/20a80092-9ee4-45f2-9a6c-a37a4b850e5c)

**Comment by VadimNotJustDev** [comment]
Great article about this: https://medium.com/hellogetsafe/how-we-added-an-animated-splash-screen-to-our-react-native-app-a7600eaa54e8

**Comment by khantanveer7** [comment]
https://www.youtube.com/live/Y_IaioaI2w0?si=WFRiZJNxSDRKu2sg
